packages/urlstate/useSharedState/useSharedState.test.ts

```
@@ -33,31 +39,32 @@ describe('useSharedState', () => {
           const hook1 = renderHook(() => useSharedState(state, getInitial));
 
           expect(hook1.result.current.state).toStrictEqual(initial);
           expect(stateSpy).toHaveBeenCalledTimes(1);
-          expect(stateSpy).toHaveBeenNthCalledWith(1, state, initial);
+          expect(stateSpy).toHaveBeenNthCalledWith(1, state);
         });
 
-        it('should get value from stateMap without getInitial', () => {
-          const stateMock = jest.fn() as unknown as void;
+        test('should get value from stateMap without getInitial', () => {
+          const stateMock = vi.fn() as unknown as void;
```

Biome (1.9.4)

[error] void is confusing outside a return type or a type parameter.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)